src/components/HomeContainer/Management/Ideas.ts

```diff
@@ -0,0 +1,30 @@
+import Individual from "../../../assets/individual.svg";
+import Productivity from "../../../assets/productivity.svg";
+import Implement from "../../../assets/implement.svg";
+
+export const Ideas = [
+  {
+    id: 1,
+    title: "Experienced Individuals",
+    description:
+      "Our network is made up of highly experienced professionals who are passionate about what they do.",
+    avatar: Individual,
+    altText: "Experienced Individuals",
```

Comment:
Please refer to [MDN docs on how to use alt text for accessibility](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt).

These images are purely decorative, so I believe the best accessibility per those guidelines is to use the empty string. If you disagree strongly, then the alt text should be descriptive of the image itself, not repetition of the section title. Best I can do if we add text is something like "A line drawing of person in a hoodie", "A line drawing of a toothed gear with a checkmark in the center", and "A line drawing of a bar chart and a line chart with values trending upward".

Reply:
Remember to update altText on the Ideas objects to empty string or something descriptive of the image itself